package.json

```diff
@@ -27,10 +27,12 @@
   },
   "homepage": "https://github.com/SteamingMutt/DiscordFeedback#readme",
   "devDependencies": {
-    "eslint": "^3.12.2",
-    "eslint-config-standard": "^6.2.1",
+    "eslint": "^4.1.1",
+    "eslint-config-standard": "^10.2.1",
```

Comment:
Upgrading standard can cause major style issues, unless you're sure this can be upgraded without issue, revert.

Reply:
This is something I added to the README branch forever ago, I'll double check everything works and get back too you.